draft-keytrans-mcmillion-protocol.md

```diff
@@ -82,8 +82,10 @@ properties are maintained.
 # Tree Construction
 
 KT allows clients of a service to query the keys of other clients of the same
-service. To do so, KT maintains two structures: (i) a log of each change to any
-key's value, and (ii) a set containing all of the key-version pairs that have been
+service. Keys are associated with *labels* and have *versions*. For example, for
```

Comment:
Correct. "Key" is supposed to mean "public key." What you are saying is correct, but I think that my phrasing is accurate as I say that "keys are associated with labels" not "labels map to keys."

I clarified what "key updates" are now a bit bellow to avoid confusion! (See most recent commit.)